converters/fromZigbee.js

```diff
@@ -4512,7 +4904,17 @@ const converters = {
         type: ['commandOn', 'commandOff'],
```

Comment:
Not needed, use already existing converter `command_on` and `command_off` instead